### DISCUSSION TITLE
Trouble setting up credentials with any other mailer

### DISCUSSION THREAD

**Discussion by systmfr** [postalserver/postal#2060] [Help with using Postal]
So I've followed everything step by step but I can never seem to actually login anywhere?
I tried creating my own credentials and login onto Ultramailer and HeartSender but they both give me an error, even with the login already provided.
Sending emails works perfectly on the platform no problem though? SPF, DKIM, MX and Return Path are all marked green.

https://gyazo.com/d77406a44ac29c8dc9c30a986914b958
https://gyazo.com/4d6cebf6a4b4b3357ed7c61acdb2ac11
https://gyazo.com/21e2d5f155c585a806b19325e73b4c63

Please suggests any commands I should try running or anything I need to recheck in the set up

**Comment by willpower232** [comment]
You could verify that your server has port 25 open using telnet from a different computer and if it is, you can have a look at the logs with `postal logs smtp` to see what commands your Postal server is receiving from this software.

**Reply by systmfr** [reply]
So I ended up changing the port to 587 so that's why it's listed below. I also have no idea what I'm looking at but I ran a test sample on the postal platform first to show that everything does work and right after I attempted to login with the correct credentials on the mailer.
Now that I've changed the port I get this message instead of the failure to send email: https://gyazo.com/4a62584149ce631608776e9137bc72ff
"server does not support secure connections"

Hopefully that helps coming up with something


The IPs listed below were all changed and randomized from what was actually displayed and sorry it's so long I didn't know what you need to see no homo
__

1. #root@mail:~# postal logs smtp
2. Attaching to postal_smtp_1
3. smtp_1      | [9] [2022-07-08T16:58:20.995] INFO -- : Listening on  :::587
4. smtp_1      | [9] [2022-07-08T17:00:47.864] DEBUG -- : [ABU9NF]    Connection opened from ::ffff:80.100.100.108
5. smtp_1      | [9] [2022-07-08T17:00:47.864] DEBUG -- : [ABU9NF]    Client identified as ::ffff:80.100.100.108
6. smtp_1      | [9] [2022-07-08T17:00:48.191] DEBUG -- : [ABU9NF] <= EHLO DESKTOP-UUQ7H9H
7. smtp_1      | [9] [2022-07-08T17:00:48.196] DEBUG -- : [ABU9NF] => 250-My capabilities are
8. smtp_1      | [9] [2022-07-08T17:00:48.196] DEBUG -- : [ABU9NF] => 250 AUTH CRAM-MD5 PLAIN LOGIN
9. smtp_1      | [9] [2022-07-08T17:00:48.611] DEBUG -- : [ABU9NF]    Connection closed
10. smtp_1      | [9] [2022-07-08T17:01:03.157] DEBUG -- : [UQMPIO]    Connection opened from ::ffff:80.100.100.108
11. smtp_1      | [9] [2022-07-08T17:01:03.157] DEBUG -- : [UQMPIO]    Client identified as ::ffff:80.100.100.108
12. smtp_1      | [9] [2022-07-08T17:01:03.473] DEBUG -- : [UQMPIO] <= EHLO DESKTOP-UUQ
13. smtp_1      | [9] [2022-07-08T17:01:03.479] DEBUG -- : [UQMPIO] => 250-My capabilities are
14. smtp_1      | [9] [2022-07-08T17:01:03.479] DEBUG -- : [UQMPIO] => 250 AUTH CRAM-MD5 PLAIN LOGIN
15. smtp_1      | [9] [2022-07-08T17:01:03.807] DEBUG -- : [UQMPIO] <= AUTH login [redacted]
16. smtp_1      | [9] [2022-07-08T17:01:03.807] DEBUG -- : [UQMPIO] => 334 UGFzc3dvcmZ3
17. smtp_1      | [9] [2022-07-08T17:01:03.939] DEBUG -- : [UQMPIO] <= [redacted]
18. smtp_1      | [9] [2022-07-08T17:01:04.359] DEBUG -- : [UQMPIO]    WARN: AUTH failure for ::ffff:80.100.100.108
19. smtp_1      | [9] [2022-07-08T17:01:04.360] DEBUG -- : [UQMPIO] => 535 Invalid credential
20. smtp_1      | [9] [2022-07-08T17:01:04.515] DEBUG -- : [UQMPIO] <= MAIL FROM:<test@myservername.com>
21. smtp_1      | [9] [2022-07-08T17:01:04.516] DEBUG -- : [UQMPIO] => 250 OK
22. smtp_1      | [9] [2022-07-08T17:01:04.648] DEBUG -- : [UQMPIO] <= RCPT TO:<smytestmail@gmail.com>
23. smtp_1      | [9] [2022-07-08T17:01:04.729] DEBUG -- : [UQMPIO] => 530 Authentication required
24. smtp_1      | [9] [2022-07-08T17:01:04.881] DEBUG -- : [UQMPIO]    Connection closed
25. smtp_1      | [9] [2022-07-08T17:01:15.667] DEBUG -- : [NYLFAI]    Connection opened from ::ffff:80.100.100.108
26. smtp_1      | [9] [2022-07-08T17:01:15.667] DEBUG -- : [NYLFAI]    Client identified as ::ffff:80.100.100.108
27. smtp_1      | [9] [2022-07-08T17:01:15.991] DEBUG -- : [NYLFAI] <= EHLO DESKTOP-UUQ
28. smtp_1      | [9] [2022-07-08T17:01:15.997] DEBUG -- : [NYLFAI] => 250-My capabilities are
29. smtp_1      | [9] [2022-07-08T17:01:15.997] DEBUG -- : [NYLFAI] => 250 AUTH CRAM-MD5 PLAIN LOGIN
30. smtp_1      | [9] [2022-07-08T17:01:16.319] DEBUG -- : [NYLFAI]    Connection closed
31. smtp_1      | [9] [2022-07-09T05:44:09.779] DEBUG -- : [ZFGBYQ]    Connection opened from ::ffff:80.100.100.108
32. smtp_1      | [9] [2022-07-09T05:44:09.784] DEBUG -- : [ZFGBYQ]    Client identified as ::ffff:80.100.100.108
33. smtp_1      | [9] [2022-07-09T05:44:09.803] DEBUG -- : [ZFGBYQ] <= EHLO www.censys.io
34. smtp_1      | [9] [2022-07-09T05:44:09.859] DEBUG -- : [ZFGBYQ] => 250-My capabilities are
35. smtp_1      | [9] [2022-07-09T05:44:09.859] DEBUG -- : [ZFGBYQ] => 250 AUTH CRAM-MD5 PLAIN LOGIN
36. smtp_1      | [9] [2022-07-09T05:44:09.874] DEBUG -- : [ZFGBYQ] <= STARTTLS
37. smtp_1      | [9] [2022-07-09T05:44:09.876] DEBUG -- : [ZFGBYQ] => 502 TLS not available
38. smtp_1      | [9] [2022-07-09T05:44:09.883] DEBUG -- : [ZFGBYQ]    Connection closed
39. smtp_1      | [9] [2022-07-09T12:17:52.810] DEBUG -- : [OMW3SY]    Connection opened from ::ffff:80.100.100.108
40. smtp_1      | [9] [2022-07-09T12:17:52.822] DEBUG -- : [OMW3SY]    Client identified as ::ffff:80.100.100.108
41. smtp_1      | [9] [2022-07-09T12:17:52.860] DEBUG -- : [OMW3SY] <= EHLO www.censys.io
42. smtp_1      | [9] [2022-07-09T12:17:52.956] DEBUG -- : [OMW3SY] => 250-My capabilities are
43. smtp_1      | [9] [2022-07-09T12:17:52.957] DEBUG -- : [OMW3SY] => 250 AUTH CRAM-MD5 PLAIN LOGIN
44. smtp_1      | [9] [2022-07-09T12:17:52.986] DEBUG -- : [OMW3SY] <= STARTTLS
45. smtp_1      | [9] [2022-07-09T12:17:52.987] DEBUG -- : [OMW3SY] => 502 TLS not available
46. smtp_1      | [9] [2022-07-09T12:17:53.002] DEBUG -- : [OMW3SY]    Connection closed
47. smtp_1      | [9] [2022-07-09T13:18:52.057] DEBUG -- : [GOWNIT]    Connection opened from ::ffff:80.100.100.108
48. smtp_1      | [9] [2022-07-09T13:18:52.058] DEBUG -- : [GOWNIT]    Client identified as ::ffff:80.100.100.108
49. smtp_1      | [9] [2022-07-09T13:18:52.060] DEBUG -- : [GOWNIT] <= GET / HTTP/1.1
50. smtp_1      | [9] [2022-07-09T13:18:52.063] DEBUG -- : [GOWNIT] => 502 Invalid/unsupported command
51. smtp_1      | [9] [2022-07-09T13:18:52.064] DEBUG -- : [GOWNIT] <= Host: 101.201.201.01:587
52. smtp_1      | [9] [2022-07-09T13:18:52.064] DEBUG -- : [GOWNIT] => 502 Invalid/unsupported command
53. smtp_1      | [9] [2022-07-09T13:18:52.064] DEBUG -- : [GOWNIT] <= User-Agent: https://gdnplus.com:Gather Analyze Provide.
54. smtp_1      | [9] [2022-07-09T13:18:52.065] DEBUG -- : [GOWNIT] => 502 Invalid/unsupported command
55. smtp_1      | [9] [2022-07-09T13:18:52.065] DEBUG -- : [GOWNIT] <=
56. smtp_1      | [9] [2022-07-09T13:18:52.066] DEBUG -- : [GOWNIT] => 502 Invalid/unsupported command
57. smtp_1      | [9] [2022-07-09T13:18:52.169] DEBUG -- : [GOWNIT]    Connection closed
58. smtp_1      | [9] [2022-07-09T16:22:03.298] DEBUG -- : [DVALNI]    Connection opened from ::ffff:80.100.100.108
59. smtp_1      | [9] [2022-07-09T16:22:03.303] DEBUG -- : [DVALNI]    Client identified as ::ffff:80.100.100.108
60. smtp_1      | [9] [2022-07-09T16:22:03.531] DEBUG -- : [DVALNI] <= EHLO DESKTOP-UUQ
61. smtp_1      | [9] [2022-07-09T16:22:03.628] DEBUG -- : [DVALNI] => 250-My capabilities are
62. smtp_1      | [9] [2022-07-09T16:22:03.629] DEBUG -- : [DVALNI] => 250 AUTH CRAM-MD5 PLAIN LOGIN
63. smtp_1      | [9] [2022-07-09T16:22:03.842] DEBUG -- : [DVALNI]    Connection closed
64. smtp_1      | [9] [2022-07-09T16:26:24.894] DEBUG -- : [HPJDMM]    Connection opened from ::ffff:80.100.100.108
65. smtp_1      | [9] [2022-07-09T16:26:24.895] DEBUG -- : [HPJDMM]    Client identified as ::ffff:80.100.100.108
66. smtp_1      | [9] [2022-07-09T16:26:25.136] DEBUG -- : [HPJDMM] <= EHLO DESKTOP-UUQ
67. smtp_1      | [9] [2022-07-09T16:26:25.155] DEBUG -- : [HPJDMM] => 250-My capabilities are
68. smtp_1      | [9] [2022-07-09T16:26:25.156] DEBUG -- : [HPJDMM] => 250 AUTH CRAM-MD5 PLAIN LOGIN
69. smtp_1      | [9] [2022-07-09T16:26:25.388] DEBUG -- : [HPJDMM]    Connection closed
70. smtp_1      | [9] [2022-07-09T21:31:34.626] DEBUG -- : [IK68SI]    Connection opened from ::ffff:80.100.100.108
71. smtp_1      | [9] [2022-07-09T21:31:34.636] DEBUG -- : [IK68SI]    Client identified as ::ffff:80.100.100.108
72. smtp_1      | [9] [2022-07-09T21:31:34.646] DEBUG -- : [IK68SI] <= EHLO zg-0421e-180
73. smtp_1      | [9] [2022-07-09T21:31:34.746] DEBUG -- : [IK68SI] => 250-My capabilities are
74. smtp_1      | [9] [2022-07-09T21:31:34.747] DEBUG -- : [IK68SI] => 250 AUTH CRAM-MD5 PLAIN LOGIN
75. smtp_1      | [9] [2022-07-09T21:31:34.748] DEBUG -- : [IK68SI] <= QUIT
76. smtp_1      | [9] [2022-07-09T21:31:34.749] DEBUG -- : [IK68SI] => 221 Closing Connection
77. smtp_1      | [9] [2022-07-09T21:31:34.749] DEBUG -- : [IK68SI]    Connection closed
78. smtp_1      | [9] [2022-07-10T03:22:40.491] DEBUG -- : [LUSBDF]    Connection opened from ::ffff:80.100.100.108
79. smtp_1      | [9] [2022-07-10T03:22:40.502] DEBUG -- : [LUSBDF]    Client identified as ::ffff:80.100.100.108
80. smtp_1      | [9] [2022-07-10T03:22:40.514] DEBUG -- : [LUSBDF] <= MGLNDD_101.201.201.0_587
81. smtp_1      | [9] [2022-07-10T03:22:40.519] DEBUG -- : [LUSBDF] => 502 Invalid/unsupported command
82. smtp_1      | [9] [2022-07-10T03:22:40.587] DEBUG -- : [LUSBDF]    Connection closed
83. smtp_1      | [9] [2022-07-10T07:33:55.058] DEBUG -- : [ZUFDJE]    Connection opened from ::ffff:80.100.100.108
84. smtp_1      | [9] [2022-07-10T07:33:55.065] DEBUG -- : [ZUFDJE]    Client identified as ::ffff:80.100.100.108
85. smtp_1      | [9] [2022-07-10T07:33:55.184] DEBUG -- : [ZUFDJE]    Connection closed
86. smtp_1      | [9] [2022-07-10T07:33:55.280] DEBUG -- : [QMP0PZ]    Connection opened from ::ffff:80.100.100.108
87. smtp_1      | [9] [2022-07-10T07:33:55.280] DEBUG -- : [QMP0PZ]    Client identified as ::ffff:80.100.100.108
88. smtp_1      | [9] [2022-07-10T07:33:55.492] DEBUG -- : [QMP0PZ] <= helo mail.myservername.com
89. smtp_1      | [9] [2022-07-10T07:33:55.598] DEBUG -- : [QMP0PZ] => 250 mail.myservername.com
90. smtp_1      | [9] [2022-07-10T07:33:55.703] DEBUG -- : [QMP0PZ] <= mail from: <M8zCIgDHQmnrU0xigM57pGtp0@mail.myservername.com>
91. smtp_1      | [9] [2022-07-10T07:33:55.707] DEBUG -- : [QMP0PZ] => 250 OK
92. smtp_1      | [9] [2022-07-10T07:33:55.811] DEBUG -- : [QMP0PZ] <= rcpt to: <whizzy000@hotmail.com>
93. smtp_1      | [9] [2022-07-10T07:33:56.089] DEBUG -- : [QMP0PZ] => 530 Authentication required
94. smtp_1      | [9] [2022-07-10T07:33:56.195] DEBUG -- : [QMP0PZ] <= quit
95. smtp_1      | [9] [2022-07-10T07:33:56.195] DEBUG -- : [QMP0PZ] => 221 Closing Connection
96. smtp_1      | [9] [2022-07-10T07:33:56.195] DEBUG -- : [QMP0PZ]    Connection closed
97. smtp_1      | [9] [2022-07-10T09:56:17.838] DEBUG -- : [DVMEFZ]    Connection opened from ::ffff:80.100.100.108
98. smtp_1      | [9] [2022-07-10T09:56:17.850] DEBUG -- : [DVMEFZ]    Client identified as ::ffff:80.100.100.108
99. smtp_1      | [9] [2022-07-10T09:56:18.091] DEBUG -- : [DVMEFZ] <= EHLO masscan
100. smtp_1      | [9] [2022-07-10T09:56:18.184] DEBUG -- : [DVMEFZ] => 250-My capabilities are
101. smtp_1      | [9] [2022-07-10T09:56:18.184] DEBUG -- : [DVMEFZ] => 250 AUTH CRAM-MD5 PLAIN LOGIN
102. smtp_1      | [9] [2022-07-10T09:56:18.416] DEBUG -- : [DVMEFZ] <= STARTTLS
103. smtp_1      | [9] [2022-07-10T09:56:18.416] DEBUG -- : [DVMEFZ] => 502 TLS not available
104. smtp_1      | [9] [2022-07-10T09:56:18.528] DEBUG -- : [DVMEFZ]    Connection closed
105. smtp_1      | [9] [2022-07-10T15:43:35.364] DEBUG -- : [SWKNEW]    Connection opened from ::ffff:80.100.100.108
106. smtp_1      | [9] [2022-07-10T15:43:35.375] DEBUG -- : [SWKNEW]    Client identified as ::ffff:80.100.100.108
107. smtp_1      | [9] [2022-07-10T15:43:35.914] DEBUG -- : [SWKNEW] <= EHLO DESKTOP-UUQ
108. smtp_1      | [9] [2022-07-10T15:43:36.037] DEBUG -- : [SWKNEW] => 250-My capabilities are
109. smtp_1      | [9] [2022-07-10T15:43:36.037] DEBUG -- : [SWKNEW] => 250 AUTH CRAM-MD5 PLAIN LOGIN
110. smtp_1      | [9] [2022-07-10T15:43:36.342] DEBUG -- : [SWKNEW]    Connection closed
111. smtp_1      | [9] [2022-07-10T15:47:12.402] DEBUG -- : [AQM0TY]    Connection opened from ::ffff:80.100.100.108
112. smtp_1      | [9] [2022-07-10T15:47:12.403] DEBUG -- : [AQM0TY]    Client identified as ::ffff:80.100.100.108
113. smtp_1      | [9] [2022-07-10T15:47:12.621] DEBUG -- : [AQM0TY] <= EHLO DESKTOP-UUQ
114. smtp_1      | [9] [2022-07-10T15:47:12.632] DEBUG -- : [AQM0TY] => 250-My capabilities are
115. smtp_1      | [9] [2022-07-10T15:47:12.632] DEBUG -- : [AQM0TY] => 250 AUTH CRAM-MD5 PLAIN LOGIN
116. smtp_1      | [9] [2022-07-10T15:47:12.847] DEBUG -- : [AQM0TY]    Connection closed
117. smtp_1      | [9] [2022-07-10T15:54:26.483] DEBUG -- : [ORJZ4B]    Connection opened from ::ffff:80.100.100.108
118. smtp_1      | [9] [2022-07-10T15:54:26.484] DEBUG -- : [ORJZ4B]    Client identified as ::ffff:80.100.100.108
119. smtp_1      | [9] [2022-07-10T15:54:26.699] DEBUG -- : [ORJZ4B] <= EHLO DESKTOP-UUQ
120. smtp_1      | [9] [2022-07-10T15:54:26.709] DEBUG -- : [ORJZ4B] => 250-My capabilities are
121. smtp_1      | [9] [2022-07-10T15:54:26.709] DEBUG -- : [ORJZ4B] => 250 AUTH CRAM-MD5 PLAIN LOGIN
122. smtp_1      | [9] [2022-07-10T15:54:26.937] DEBUG -- : [ORJZ4B]    Connection closed
123. smtp_1      | [9] [2022-07-10T15:56:28.390] DEBUG -- : [BOLBHI]    Connection opened from ::ffff:80.100.100.108
124. smtp_1      | [9] [2022-07-10T15:56:28.390] DEBUG -- : [BOLBHI]    Client identified as ::ffff:80.100.100.108
125. smtp_1      | [9] [2022-07-10T15:56:28.617] DEBUG -- : [BOLBHI] <= EHLO DESKTOP-UUQ
126. smtp_1      | [9] [2022-07-10T15:56:28.627] DEBUG -- : [BOLBHI] => 250-My capabilities are
127. smtp_1      | [9] [2022-07-10T15:56:28.627] DEBUG -- : [BOLBHI] => 250 AUTH CRAM-MD5 PLAIN LOGIN
128. smtp_1      | [9] [2022-07-10T15:56:28.853] DEBUG -- : [BOLBHI]    Connection closed
129. root@mail:~#
130.

**Reply by willpower232** [reply]
> So I ended up changing the port to 587

Postal only supports explicit TLS not implicit TLS so its possible your software starts talking in TLS before asking Postal to start talking in TLS.

**Reply by systmfr** [reply]
So I've been able to send mail by checking off the "enable SSL/TLS" box in the UltraMailer software but whenever it's enabled I get the same "server does not support secure connections". What could I do to fix this? I read what you said above are you saying to try a different program?

**Reply by willpower232** [reply]
Your software might expect TLS to start happening automatically if it is using port 587, you should probably stick to port 25 and then hopefully your software will ask about upgrading to TLS explicitly.
